docs/guide/resources.md

```diff
@@ -19,6 +19,7 @@
 | [Biome's noReExportAll](https://biomejs.dev/linter/rules/no-re-export-all/) | Biome plugin to avoid re-export all |
 | [unplugin-purge-polyfills](https://github.com/danielroe/unplugin-purge-polyfills) | Unplugin to replace package imports with native code |
 | [Package Size Calculator](https://github.com/TheDevMinerTV/package-size-calculator) | CLI to calculate the size of a package after removing/replacing dependencies, or calculating the difference of two versions of the same package |
+| [howfat](https://github.com/megahertz/howfat) | Shows how fat is a package together with its dependencies |
```

Comment:
Can we correct the English a little? (I know it's written this way at the source repo too)

> Shows how much space a package takes up on disk including dependencies

Reply:
done